DependencyInjection/CmfSeoExtension.php

```diff
@@ -90,6 +92,10 @@ public function load(array $configs, ContainerBuilder $container)
         if ($this->isConfigEnabled($container, $config['alternate_locale'])) {
             $this->loadAlternateLocaleProvider($config['alternate_locale'], $container);
         }
+
+        if ($this->isConfigEnabled($container, $config['form'])) {
```

Comment:
if we drop the canBeDisabled this could go and we would just load unconditionally.